anyway/widgets/widget_utils.py

```diff
@@ -173,14 +173,29 @@ def gen_entity_labels(entity: Type[LabeledCode]) -> dict:
     return res
 
 
-def get_injured_filters(location_info):
-    new_filters = {}
-    for curr_filter, curr_values in location_info.items():
+def get_involved_marker_view_location_filters(
+    resolution: BE_CONST.ResolutionCategories, location_info: LocationInfo
+):
+    filters = {}
+    if resolution == BE_CONST.ResolutionCategories.STREET:
+        filters["involve_yishuv_name"] = location_info.get("yishuv_name")
+        filters["street1_hebrew"] = location_info.get("street1_hebrew")
+    elif resolution == BE_CONST.ResolutionCategories.SUBURBAN_ROAD:
+        filters["road1"] = location_info.get("road1")
+        filters["road_segment_name"] = location_info.get("road_segment_name")

```

Comment:
Should we handle additional resolutions here, or not yet?

Reply:
When I got the first task of relocating widgets to AllLocation, Atalya told me to implement it for road+street resolutions, and we will ask about it. We can go ahead with it, but what are the identifiers in the news_flash table for non-urban intersections? Is 'non_urban_intersection_hebrew' sufficient, or should it be 'non_urban_intersection_hebrew' and 'road_segment_name'?

Reply:
OK then. Just wanted to verify

Reply:
@atalyaalon 
FYI

Reply:
The current situation is OK, in the coming future I'll open a separate issue regarding additional resolutions